wrappers/net/http/client_test.go

```diff
@@ -260,6 +310,24 @@ var _ = Describe("ClientWrapper", func() {
 				verifyTraceIDExists(events[0])
 			})
 		})
+		Context("sending a request to existing server, no tracer", func() {
+			It("adds an event with no error", func() {
+				tracer.GlobalTracer = nil
+				client := Wrap(http.Client{})
+				response, err := client.PostForm(
+					testServer.URL,
+					map[string][]string{
+						"hello": []string{"world", "of", "serverless"},
+					},
+				)
+				Expect(err).To(BeNil())
+				defer response.Body.Close()
+				responseData, err := ioutil.ReadAll(response.Body)
+				Expect(err).To(BeNil())
+				responseString := string(responseData)
+				Expect(responseString).To(Equal(TEST_RESPONSE_STRING))
```

Comment:
This block of code 
Expect(err).To(BeNil())
				defer response.Body.Close()
				responseData, err := ioutil.ReadAll(response.Body)
				Expect(err).To(BeNil())
				responseString := string(responseData)
				Expect(responseString).To(Equal(TEST_RESPONSE_STRING))
returns multiple times, maybe its better to put it inside a function